.travis.yml

```diff
@@ -15,5 +15,8 @@ cache:
   - pip
 addons:
   postgresql: '9.6'
+notifications:
+  email:
+    mike.ubell@gmail.com
```

Comment:
This should start with a dash so it's an array:
https://docs.travis-ci.com/user/notifications/#configuring-email-notifications

```yaml
notifications:
  email:
    - mike.ubell@gmail.com
```